src/extensions.js

```diff
@@ -0,0 +1,266 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//
+// Extension parsing.
+//
+// This file contains some helpers reading/writing the various kinds
```

Comment:
`some helpers reading/writing` -> `some helpers for reading/writing` ?

Reply:
also this comment didn't make it to the final bundle, need to regenerate the build